angr/analyses/decompiler/structurer.py

```diff
@@ -672,6 +672,11 @@ def _make_ites(self, seq):
                     new_node = ConditionNode(node.addr, None, node.reaching_condition, node,
                                              None)
                 seq.nodes[i] = new_node
+            else:
+                if isinstance(node, CodeNode) and isinstance(node.node, ConditionalBreakNode):
```

Comment:
Can we do an `elif:` instead of `else:\nif ...`?

Reply:
Done